decko-rails/decko-rails.gemspec

```diff
@@ -5,13 +5,12 @@ version = File.open(File.expand_path( '../../card/VERSION', __FILE__ )).read.cho
 Gem::Specification.new do |s|
   s.name          = 'decko-rails'
   s.version       = version
```

Comment:
I pushed decko-rails spec to rubygems for the first time today (made sure we had the name)